src/modules/systemlib/system_params.c

```diff
@@ -134,7 +134,7 @@ PARAM_DEFINE_INT32(SYS_MC_EST_GROUP, 2);
  * @value 1921600 ESP8266 (921600 baud, 8N1)
  *
  * @min 0
- * @max 1921600
+ * @max 6460800
```

Comment:
@dagar I am not sure if this is actually the right way of doing it, but if you remove the first digit from each baud rate number, then yes `921600` is the max parameter value. But having the first digit, isn't `6460800` the max value then?

Reply:
The first digit is simply for the parameter. Take a look at how it's used in the init.
https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/init.d/rcS#L644-L708

Reply:
So what should be considered the "max" value? `1921600` or `921600`?

Reply:
With enums you can drop min and max entirely.

Reply:
So why do we have it here in the first place? Should `max` and `min` be taken?

Reply:
I'm guessing at one point the parameter value itself was used directly as the baud rate. Yes you can drop min and max, the only values shown in QGC will be the enums (@value).